README.rst

```diff
@@ -112,6 +112,11 @@ asset by editing the asset in Superset and selecting "Save As" to save it to a n
 # Note: If you are using custom assets you will need to rebuild your aspects-superset
 # image on your local machine with `tutor images build aspects-superset --no-cache`.
 
+Assets created for Aspects that are no longer used are specified in `aspects_asset_list.yaml`
+and will be deleted from the Superset database when import-assets is run.
```

Comment:
We should add that `import-assets` gets run as part of init, since that's the way most people would interact with this.